README.md

```diff
@@ -83,7 +83,7 @@ If you need to build Julia in an environment that does not allow access to the o
 **Note:** the build process fail badly if any of the build directory's parent directories have spaces or other shell meta-characters such as `$` or `:` in their names (this is due to a limitation in GNU make).
 
 Once it is built, you can run the `julia` executable using its full path in the directory created above (the `julia` directory), or, to run it from anywhere, either
-
+- add an alias (in `bash`: ` echo "alias julia='/path/to/install/folder/usr/bin/julia'" >> ~/.bashrc && source ~/.bashrc`), or
```

Comment:
That's not consistency - `/usr/local/bin` is a location that is often on path by default, which isn't relevant for the alias method. A folder `usr/` will only be part of the path to a Julia executable in a source build that you haven't yet run `make install` on. An install tree or binary will not have `usr/` in its structure.